README.md

```diff
@@ -20,6 +26,9 @@ See [Changes.md](Changes.md) for a complete list of features, fixes, and changes
 
 
 # Requirements
+
+`bs-platform` 6.2.0 or higher: use `gentype` 3.0.0 or higher.
+
 `bs-platform` 5.2.0 or higher: use `genType` 2.40.0 or higher.
```

Comment:
Should also say to use the other branch.

Reply:
this information would only be relevant for contributors, right? I thought if we keep this leaner it's easier to understand for users

Reply:
This is the main README, and people who wish to build from source would know where to find it.